presto-prometheus/src/main/java/io/prestosql/plugin/prometheus/PrometheusConnectorConfig.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.prometheus;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.ConfigurationException;
+import com.google.inject.spi.Message;
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.units.Duration;
+import io.airlift.units.MinDuration;
+
+import javax.annotation.PostConstruct;
+import javax.validation.constraints.NotNull;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+public class PrometheusConnectorConfig
+{
+    private URI prometheusURI = new URI("http://localhost:9090");
+    private Duration queryChunkSizeDuration = new Duration(1, TimeUnit.DAYS);
+    private Duration maxQueryRangeDuration = new Duration(21, TimeUnit.DAYS);
+    private Duration cacheDuration = new Duration(30, TimeUnit.SECONDS);
+    private String bearerTokenFile;
+
+    public PrometheusConnectorConfig()
+            throws URISyntaxException
+    {}
+
+    @NotNull
+    public URI getPrometheusURI()
+    {
+        return prometheusURI;
+    }
+
+    @Config("prometheus-uri")
+    @ConfigDescription("Where to find Prometheus coordinator host")
+    public PrometheusConnectorConfig setPrometheusURI(URI prometheusURI)
+    {
+        this.prometheusURI = prometheusURI;
+        return this;
+    }
+
+    @MinDuration("1ms")
+    public Duration getQueryChunkSizeDuration()
+    {
+        return queryChunkSizeDuration;
+    }
+
+    public String getQueryChunkSizeDurationAsPrometheusCompatibleDurationString()
```

Comment:
Please remove (or move to test) this method since it's called only from the test.

Reply:
Done in dc0a1bdf